src/scripts/migrate-traits.js

```diff
@@ -0,0 +1,238 @@
+    console.log("Loading members: " + searchResults.hits.hits.length + " of: " + body.hits.total)
+    
+    // collect the titles from this response
+    body.hits.hits.forEach(function (hit) {
+      searchResults.hits.hits.push(hit)
+    })
+
+    // check to see if we have collected all of the quotes
+    if (body.hits.total === searchResults.hits.hits.length) {
+      searchResults.hits.total=body.hits.total
+      break
+    }
+
+    // get the next response if there are more quotes to fetch
+    responseQueue.push(
+      await esClient.scroll({
+        scroll_id: body._scroll_id,
+        scroll: '90s'
+      })
+    )
+  }
+  return searchResults.hits.hits || []
+}
+
+async function migrateMemberData(member, applyForReal) {
+    let skillScoreDeduction = 0
+    let workHistory = false
+    let education = false
+    const dbMember = await helper.getMemberByHandle(member.handle)
+
+    let education_trait = _.find(member.traits, function(trait){ return trait.traitId == "education"})
+    console.log(member.handle, "Found education trait", JSON.stringify(education_trait, null, 5))
+    if(education_trait && education_trait.traits.data && education == false){
+      education = true
+    }
+  
+    let work_trait = _.find(member.traits, function(trait){ return trait.traitId == "work"})
+    console.log(member.handle, "Found work trait", JSON.stringify(education_trait, null, 5))
+    if(work_trait && work_trait.traits.data && workHistory==false){
+      workHistory = true
+    }
+  
+    let personalization_trait = _.find(member.traits, function(trait){ return trait.traitId == "personalization"})
+    console.log(member.handle, "Found personalization trait", JSON.stringify(personalization_trait, null, 5))
+    if(personalization_trait && personalization_trait.traits.data && personalization_trait.traits.data.namesAndHandleAppearance){
+        dbMember.namesAndHandleAppearance = personalization_trait.traits.data.namesAndHandleAppearance
+    }
+
+    if(personalization_trait && personalization_trait.traits.data && personalization_trait.traits.data.availableForGigs){
+        dbMember.availableForGigs = personalization_trait.traits.data.availableForGigs
+    }
+  
+    // TAL-77 : missing experience, reduce match by 2%
+    if(!workHistory){
+      skillScoreDeduction = skillScoreDeduction - 0.02
+    }
+  
+    // TAL-77 : missing education, reduce match by 2%
+    if(!education){
+      skillScoreDeduction = skillScoreDeduction - 0.02
+    }
+    
+    dbMember.skillScoreDeduction = skillScoreDeduction
+    
+    console.log(dbMember.handle, "Skill score deduction", skillScoreDeduction)
+    console.log(dbMember.handle, "Names and handle appearance", dbMember.namesAndHandleAppearance)
+    console.log(dbMember.handle, "Available for gigs", dbMember.availableForGigs)
+    if(applyForReal){
+        try{
+            const result = await helper.update(dbMember, {"skillScoreDeduction":skillScoreDeduction})
```

Comment:
Need a space here after the colon: `{"skillScoreDeduction": skillScoreDeduction}` 

Also, for JS code, do we not typically add spaces between curly braces and their contained contents? Other languages do, I just don't recall if we use that style here.